filer/static/filer/js/base.js

```diff
@@ -43,29 +43,74 @@ var Cl = window.Cl || {};
             }
         }());
 
-        // mocking the action buttons to work in frontend UI, please do not review
+        // show counter if file is selected
+        (function () {
+            var table = $('.navigator-table').find('tr');
+            var actionCounter = $('.action-counter');
+            var actionSelect = $('.action-select, .column-checkbox');
+
+            actionSelect.each(function () {
+                $(this).on('click', function () {
```

Comment:
also, it should not be a click, because you can change checkbox by other means (e.g keyboard)